### PR TITLE
Drop RPos and SPos from `map_conv_to_matmul`

### PR DESCRIPTION
**Pull request by chelini**
In the case of the sliding window, we need to read the sizes from the filter and the output. If we don't have a sliding window, we read the sizes from the operand, but numerically, they always agree with the sliding window case.

### CODE REVIEW

**Comment by chelini** [comment]
> It seems to me that R&S pos weren't being used at all other than verifying the contiguous constraints. If that's true, why did we have that constraint in the first place? And how are we going to know where R and S are? Can we safely guess every time?

The sizes that we need to extract from the image are numerically equivalent if we have a sliding window or not so we can simply avoid R and S, and get them using `computeSizeGemmForImage` this should be correct in all cases.

File: include/TPP/Transforms.h [hunk 1]
```diff
@@ -29,12 +29,12 @@ namespace linalgx {
 FailureOr<SmallVector<Value>> mapToBRGEMMOp(RewriterBase &rewriter,
                                             linalg::LinalgOp linalgOp);
 
+// TODO: this is now more generic.
```
[the remark this answers is not in the document]
Comment (chelini) [reply include/TPP/Transforms.h:32]: Yeah, my bad. Just ignore. It is just more generic and I need to rename it.

[the remark this answers is not in the document]
Comment (rengolin) [reply include/TPP/Transforms.h:32]: right, then just remove the "TODO" part and it should be fine.

[the remark this answers is not in the document]
Comment (chelini) [reply include/TPP/Transforms.h]: Dropped.

**Review by rengolin — APPROVED** [review]

**Pull request merged**